mlflow/store/artifact/artifact_repo.py

```diff
@@ -145,6 +145,19 @@ def list_artifacts(self, path):
             List of artifacts as FileInfo listed directly under path.
         """
 
+    def list_logged_model_artifacts(self, path: Optional[str] = None) -> List[FileInfo]:
```

Comment:
In databricks artifact repo, we didn't define separate methods for uploading / downloading artifacts. Should we be consistent for listing as well?

Reply:
Ok, we can just remove this method and use list_artifacts?